php_phongo.c

```diff
@@ -177,6 +177,10 @@ PHP_MINIT_FUNCTION(mongodb) /* {{{ */
 	 * a logger. */
 	mongoc_log_set_handler(NULL, NULL);
 
+	/* Disable trace logging. This will be enabled in phongo_log_sync_handlers()
+	 * if the "mongodb.debug" INI option is set. */
+	mongoc_log_trace_disable();
```

Comment:
I confirmed that MINIT is called _before_ OnUpdateDebug, so this doesn't conflict with tracing enabled via INI files.